definitions/variable/macro-economy/price.yaml

```
@@ -0,0 +1,26 @@
+- Price|Carbon:
+    description: Price of carbon (as reported by the model)
+    unit: USD_2010/t CO2
+    region-aggregation:
+      - Price|Carbon (mean):
+          method: mean
+      - Price|Carbon (weighted by Emissions|CO2):
+          weight: Emissions|CO2
+          drop_negative_weights: false
+      - Price|Carbon (weighted by Final Energy):
+          weight: Final Energy
+    check-aggregate: false
+- Price|Carbon (mean):
+    description: Price of carbon (computed as unweighted mean across regions)
+    unit: USD_2010/t CO2
+    skip-region-aggregation: true
+- Price|Carbon (weighted by Emissions|CO2):
+    description: Price of carbon (weighted by regional CO2 emissions)
+    unit: USD_2010/t CO2
+    skip-region-aggregation: true
+    note: Regional CO2 emissions can turn negative at different points in time, which
+          can result in counter-intuitive timeseries when used as weights for averages
```

Use `notes`, harmonize indent and full stop at end of sentence for consistency.
```suggestion
    notes: Regional CO2 emissions can turn negative at different points in time, which
       can result in counter-intuitive timeseries when used as weights for averages.
```

---

sounds good!
I would propose to extend the notes: 
"Regional CO2 emissions can turn negative at different points in time, which can result in counter-intuitive aggregation results when used as weights for the region-aggregation. Also, regions will high carbon prices will have lower emissions, meaning the emission-weighted average of carbon prices will be low, although the high carbon price may be applied to a large share of the global energy system. Using final energy as weight overcomes these aggregation issues."

which points to the alternative without these problems, but still doesn't say "take this as default".